console/src/main/scripts/bower.json

```diff
@@ -22,7 +22,7 @@
     "ngInfiniteScroll": "1.2.1",
     "bootstrap-select": "1.6",
     "hawkular-charts": "0.5.6",
-    "hawkular-ui-services": "0.8.2",
+    "hawkular-ui-services": "0.8.3",
```

Comment:
Services has been busy this morning, the version you want is actually 0.8.5 now.